doc/devel/documenting_mpl.rst

```diff
@@ -364,10 +252,8 @@ Examples
 --------
 
 The source of the files in the ``examples`` directory are
-automatically included in the HTML docs.  An image is generated and
-included for all examples in the ``api`` and ``pylab_examples``
-directories.  To exclude the example from having an image rendered,
-insert the following special comment anywhere in the script:
+automatically included in the HTML docs. To exclude the example from having an
+image rendered, insert the following special comment anywhere in the script:
```

Comment:
Is this still true @choldgraf ?

Reply:
ah no, instead they should put `sgskip` somewhere in the filename. Maybe worth just noting "all files in the `examples` directory will be automatically run and rendered into documentation with sphinx-gallery (incl. link)" and leave it at that?